spec/features/admin/products_spec.rb

```diff
@@ -488,7 +488,7 @@
           expect(flash_message).to eq('Product "a product" has been successfully updated!')
           product.reload
           expect(product.variant_unit).to eq(var_unit)
-          # TODO -> expect(page).to have_select('product_variant_unit_with_scale', :selected => dropdown_option)
+          expect(page).to have_select('product_variant_unit_with_scale', :selected => dropdown_option)
```

Comment:
I know we mix them but IMO we should favor the hash symbol syntax over =>. See https://stackoverflow.com/questions/8675206/is-there-any-difference-between-the-key-value-and-key-value-hash-no

Reply:
I just uncomment the code written by @filipefurtad0 but yes, I guess I could change it.
You mean:
```ruby
expect(page).to have_select('product_variant_unit_with_scale', selected: dropdown_option)
```
right?

Reply:
Actually, I had a notification from Rubocop (i guess) inside my favorite ide
<img width="958" alt="Capture d’écran 2021-04-14 à 16 13 55" src="https://user-images.githubusercontent.com/296452/114725094-83d05800-9d3c-11eb-89ba-107cc6118cc1.png">
Anyway, this is now corrected.

Reply:
Which IDE is that?

Reply:
the "new" Ruby 1.9 hash syntax is exactly `key: value` rather than `key => value`.

Reply:
> Which IDE is that?

Visual Studio Code.